.murdock

```diff
@@ -4,6 +4,7 @@
 : ${TEST_BOARDS_LLVM_COMPILE:="iotlab-m3 native nrf52dk mulle nucleo-f401re samr21-xpro slstk3402a"}
 
 export RIOT_CI_BUILD=1
+export RIOTLINKSEED=2342
```

Comment:
With ssp, compile/link time random value and now this, it would be the third user of a build time random variable. Can we maybe unify this properly?

Reply:
Sure, what about adding a `REPRODUCIBLE_BUILD` environment variable which deactivates SSP and uses a constant seed?